modules/cache_behavior/variables.tf

```diff
@@ -93,6 +93,12 @@ variable "whitelisted_cookie_names" {
   default     = []
 }
 
+variable "forward_headers" {
+  description = "Headers, if any, that you want CloudFront to vary upon for this cache behavior. Specify \"*\" to include all headers."
```

Comment:
I see now that this is just to escape the double quotes. Carry on!

Reply:
The backslashes escape the quote. There's no option for code formatting in these descriptions so instead I put it in quotes. Since the value it is expecting is a string then it doesn't make much of a different.